src/content/h1/tools/hek-plus/readme.md

```diff
@@ -0,0 +1,34 @@
+**Halo Editing Kit Plus (HEK+)** is a tool for extracting tags from Halo Custom
```

Comment:
Thoughts on using past tense for deprecated tools? “HEK+ was...”

Reply:
I think past tense is a bit strange from a wiki perspective. These tools are still out there and can be used, it's just that their usage is discouraged. I'd prefer to reserve past tense for things that are no longer available at all.

For a silly comparison, the Wikipedia page for the Byzantine Empire uses "was", while the Wiki page for E.T. on the Atari 2600 uses "is".